### DISCUSSION TITLE
Make vote button more prominent

### DISCUSSION THREAD

**Discussion by cobrexus** [community/community#4478] [Discussions] [answered]
The "vote" button in its current state is not prominent enough to actually catch users' attention and appear like a vote button. Most people continue to vote on discussions using the :+1: and :-1: emojis.

Compare now:

[![Image of current vote button in context](https://i.imgur.com/4jm2EuP.png)](https://i.imgur.com/4jm2EuP.png)

With a possible redesign:

[![Image of possible redesign](https://i.imgur.com/8oBLszH.png)](https://i.imgur.com/8oBLszH.png)

Notice how the vote button and number of votes in the second image are far more prominent, more than the emoji reactions even.

This redesign would encourage users to vote on discussion posts more.

**Comment by Zerotask** [comment]
Good idea. Additionally a thumps up could automatically apply a vote up

**Comment by rioastamal** [comment]
Agree. To be honest I did not know that those "arrow" are acutally a "Vote". There is no tooltips or indicator.

**Comment by mrousavy** [comment]
I personally don't like the idea of having two separate things - I find the thumbs up reaction friendlier and more eye-catchy, so maybe thumbs up/down could be used entirely instead of the vote buttons?

**Comment by cobrexus** [comment, marked as answer]
GitHub has now moved the vote and <kbd>Mark as answer</kbd> buttons next to the emoji button, which I think has achieved what I was looking for.

![image](https://user-images.githubusercontent.com/34424160/129904345-769385e8-4b6e-4023-9a08-8ff0e45fd7fb.png)
